coral-spark/src/main/java/com/linkedin/coral/spark/CoralToSparkSqlCallConverter.java

```diff
@@ -170,7 +170,6 @@ public CoralToSparkSqlCallConverter(Set<SparkUDFInfo> sparkUDFInfos) {
 
   @Override
   public SqlNode visit(SqlCall call) {
-    final SqlCall transformedSqlCall = sqlCallTransformers.apply(call);
```

Comment:
Why is this class modified? It looks like this class is in Coral-Spark not Coral-Trino

Reply:
Good point, I removed this change since this PR should be for Coral-Trino migration only, will include the change for future migration in Coral-Spark.